colmi_r02_client/real_time/enum.py

```diff
@@ -0,0 +1,46 @@
+from enum import IntEnum, StrEnum
+
+class Action(IntEnum):
+    START = 1
+    PAUSE = 2
+    CONTINUE = 3
+    STOP = 4
+
+class RealTimeReading(IntEnum):
+    """
+    Taken from https://colmi.puxtril.com/commands/#data-request
+    """
+    HEART_RATE = 1
+    BLOOD_PRESSURE = 2
+    SPO2 = 3
+    FATIGUE = 4
+    HEALTH_CHECK = 5
+    # leaving this out as it's redundant
+    # REAL_TIME_HEART_RATE = 6
+    ECG = 7
+    PRESSURE = 8
+    BLOOD_SUGAR = 9
+    HRV = 10
```

Comment:
initial plan was to add all of the ones found at https://colmi.puxtril.com/commands/#data-request, test each one and remove the ones that didn't work. apparently all of them do so I just removed the duplicate heart rate one :p